rosetta/create-t5x-distribution.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+set -euox pipefail
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+if [[ $# -ne 1 ]]; then
+  echo "[ERROR]: Need to pass a patchlist file: ./$0 <patchlist.txt>"
+  exit 1
+fi
+
+PATCH_LIST=$(readlink -f $1)
+
+# This is the git directory of t5x we will build the distribution in
+INSTALLED_T5X_DIR=${INSTALLED_T5X_DIR:-}
+# If provided, this is the commit that we'll base the distribution on
+DISTRIBUTION_BASE_REF=${DISTRIBUTION_BASE_REF:-}
+INTERNAL_T5X_SUBMODULE_DIR=$SCRIPT_DIR/t5x
+UPSTREAM_GIT_URL=https://github.com/google-research/t5x.git
+
+if [[ -z "$INSTALLED_T5X_DIR" ]]; then
+  echo "[ERROR]: Need to specify the env INSTALLED_T5X_DIR="
+  exit 1
+fi
+
+cd $INSTALLED_T5X_DIR
+git config user.email "jax@nvidia.com"
```

Comment:
Does this email exist? We should set it up properly so that messages sent to this address will be forwarded to us.

Reply:
I believe so. @sbhavani can confirm

Reply:
It does